living_document.qmd

```diff
@@ -20,51 +20,169 @@ Welcome to our class's collaborative guide! This living document was created by
+| `x`            | data.frame / matrix  | The data to be rendered as a table. Required.                             |
+| `format`       | character            | Output format: `"markdown"`, `"html"`, `"latex"`, etc. Usually auto-detected. |
+| `digits`       | integer              | Number of decimal places to display. Defaults to `getOption("digits")`.     |
+| `row.names`    | logical / `NA`       | Whether to include row names: `TRUE`, `FALSE`, or `NA` for auto-detect.     |
+| `col.names`    | character vector     | Custom column names for the table.                                          |
+| `align`        | character vector     | Column alignment: `"l"` (left), `"c"` (center), `"r"` (right).              |
+| `caption`      | character            | Table caption shown above the table.                                        |
+| `label`        | character            | Label used for cross-referencing (e.g., `@tbl-summary`).                    |
+| `format.args`  | list                 | Additional arguments passed to `format()`.                                  |
+| `escape`       | logical              | Escape special characters like `_`, `&`. Use `FALSE` to allow raw HTML/LaTeX. |
+| `...`          | —                    | Additional arguments passed to formatting methods.                          |
+
+**Example**
+
+```{r, warning=FALSE}
+library(knitr)
+# Create a data frame similar in structure to the markdown example
+data <- data.frame(
+  Default = c("12", "123", "1"),
+  Left = c("12", "123", "1"),
+  Right = c("12", "123", "1"),
+  Center = c("12", "123", "1")
+)
+
+# Generate the table using knitr::kable()
+# Set column alignment: left, left, right, and center
+# Add a caption to the table
+kable(
+  data,
+  align = c("l", "l", "r", "c"),
+  caption = "Table 1. Demonstration of knitr::kable() table with alignment"
+)
+
+```
+
+### Comparison: Quarto Markdown Table vs knitr::kable()
+
+| Feature | Quarto Markdown Table | `knitr::kable()` Table |
+|-----------------|----------------------------|----------------------------|
+| Syntax Style | Pure Markdown | R code chunk using `kable()` |
+| Data Source | Written manually | R data frame or matrix |
+| Captions Support | Yes (with `:...` line after the table) | Yes (`caption = "..."`) |
+| Auto Numbering | Yes | Yes |
+| Cross-referencing | Yes (e.g., `@tbl-id`) | Yes (with `label = "tbl-id"`) |
+| Output Formats | HTML, PDF, DOCX | HTML, PDF, DOCX |
+| Styling (basic) | Manual (using colons `:` for alignment) | Basic via `align = c("l", "r", ...)` |
+| Styling (advanced) | Limited | Supports `kableExtra`, `booktabs`, etc. |
+| Dynamic Content | No | Yes – generated from live R data |
+| Best For | Simple static tables or syntax demos | Data-driven tables, dynamic reports |
+
 ## Citations and Bibliography
+
 <!-- TODOs: .bib file usage, citation styles, Zotero integration -->
 
----
+------------------------------------------------------------------------
```

Comment:
You seem to have changed other sections of the document as well (e.g. spacing and adding dashes to horizontal lines). Could you reverse these edits? They will create unnecessary merge conflicts down the line.